icons/ngrx/ngrx-plain.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+  <path fill="#412846"
```

Comment:
The fill can be removed since it's already in `devicon.json`, which is used to generate the colored version
```suggestion
  <path
```